src/dags/crowdmonitor.py

```diff
@@ -28,11 +28,27 @@
 
 
 SQL_RENAME_TEMP_TABLE = """
-    DROP TABLE IF EXISTS {{ params.base_table }}_old;
```

Comment:
Geen backup tabel nodig?

Reply:
Ik zal de backup weer toevoegen. Bij anderen is er een check stap, en de rename wordt pas gedaan als de check OK is. Maar hier is (nog) geen check . Dan kan kan het geen kwaad om een backup te hebben